src/ert/run_models/base_run_model.py

```diff
@@ -574,6 +574,8 @@ async def run_ensemble_evaluator_async(
             self._end_queue.get()
             return []
         await evaluator_task
+        ensemble.refresh_ensemble_state()
```

Comment:
How slow is this? I was thinking of the risk of this coming out of sync with individual realizations of somethings fails during storing of data.

Reply:
In some edge cases maybe very slow, but it is cached, and the cache should only be invalidated when there is an explicit update done, at all placements of `ensemble.refresh_ensemble_state()`